bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/config/NAThingConfiguration.java

```diff
@@ -25,7 +25,7 @@ public class NAThingConfiguration {
     public static final String ID = "id";
 
     protected String id = "";
-    public int refreshInterval = -1;
+    public int refreshInterval = 120;
```

Comment:
Why 120, it's documented as 10 minutes, so shouldn't it be 600?

Reply:
Hmm, for `netatmo:home` default is actually 180 in the XML, so:
- 120 here
- 180 in XML
- 600 in README

Reply:
> Why 120, it's documented as 10 minutes, so shouldn't it be 600?

Because this is common to other things (Home uses a default of 180).

Reply:
Okay. Is there any scenario where this is actually used rather the value defined in XML (for home/weather)?

Reply:
Yes, with unmanaged configuration (eg things file) the default value of the object will be used.

Reply:
Okay, after much confusion, let me try to analyze/understand it from scratch. 😄 The parameter applies to the following thing types:
- `home` → config `home` → default **180** in XML
- `weather-station` → config `weather` → default **600** in XML (and README)
- `home-coach` → config `weather` → default **600** in XML (and README)

So I still don't understand the 120, when none of the above have 120 as default? And shouldn't the default be applied according to the thing/module type, so that it will be correct for unmanaged configuration also?

Sorry for being a bit persistent here, but somehow it seems inconsistent.

Reply:
That's ok being persistent, you push me in the right direction.
I adapted `NAThingConfiguration` and `HomeConfiguration` accordingly to be aligned with values defined in XML.